cmd/ltagent/init.go

```diff
@@ -208,6 +209,17 @@ func genAdmins(config *loadtest.Config, userPrefix string) error {
 		}
 	}
 
+	cpaEnabled, resp := control.CustomProfileAttributesEnabled(sysadmin)
+	if resp.Err != nil {
+		return resp.Err
+	}
+	if cpaEnabled {
+		err = loadtest.CreateCustomAttributeFields(sysadmin)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
This will create the Fields in the database. I'm expecting this code to run once and only once. Otherwise, I guess we need to add these to the database dump.